data-raw/toy_data.R

```diff
@@ -0,0 +1,9 @@
+# styler: off
+toy_data <- tibble::tribble(
+  ~bank, ~amount, ~amount_of_disctinct_products, ~PCTR_risk_category,  ~tilt_sec,
```

Comment:
FIXME: Can we stick to the tidyverse style guide and use `snake_case`? Maybe check with Anne (the offender is the uppercase `PCTR_*` prefix).